draft-ietf-acme-acme.md

```diff
@@ -1930,7 +1930,9 @@ for the domain in question.
 The path at which the resource is provisioned is comprised of the fixed prefix
 ".well-known/acme-challenge/", followed by the "token" value in the challenge.
 The value of the resource MUST be the ASCII representation of the key
-authorization.
+authorization. The resource should remain provisioned there until the client
```

Comment:
tiny editorial nit: I think "there" in this sentence reads a little funny and is redundant with the initial path sentence. Can you say "The resource should remain provisioned until the client ...." and leave it at that? If you disagree I'd let it rest as-is :-)